app/views/unattended/poap/disklayout.erb

```diff
@@ -0,0 +1,5 @@
+<%#
+kind: ptable
+name: NX-OS default fake
+%>
+# Not required for NX-OS POAP provisioning.
```

Comment:
I understand that a partition table is required for building a host, and it should be associated with the operating system family, even with the installer does not use it. There are currently two fake partition table templates, one for CoreOS and one for Junos, so I ended up using the same approach and included a dummy partition template for NX-OS.
Could you please confirm that adding a simple, dummy template to [community-templates](https://github.com/theforeman/community-templates) as a convenience to other users would be the right thing?